### PR TITLE
fix(gen): format, fix sample configs used for generating recipes

### PR DESCRIPTION
**Pull request by sudo-suhas**
- Introduce and use template helper function on sample configs of
  sinks, extractors and processors to replace tabs, if present, with
  spaces in sample configs. If tabs are present, parsing the file as
  YAML fails.
- Fix sample configs for sinks/http, extractors/{kafka, tableau}.
- Reformat sample configs for consistency.

### CODE REVIEW

**Comment by sudo-suhas** [comment]
@StewartJingga this issue is present in #408 as well. Can we review and merge this change?

File: generator/recipe.go [hunk 1]
```diff
@@ -93,6 +94,14 @@ func indent(spaces int, v string) string {
 	return pad + strings.Replace(v, "\n", "\n"+pad, -1)
 }
 
+func fmtSampleConfig(s string) string {
```
Comment (ravisuhag) [inline generator/recipe.go:97]: @sudo-suhas lets rename this to something short? maybe `cfmt` ?

Comment (sudo-suhas) [reply generator/recipe.go:97]: Does rawfmt work?

**Comment by StewartJingga** [comment]
Should we move sample config to its own file 🤔 ? or even better ideally we should be able to generate sample config from the `Config` struct itself. wdyt @ravisuhag @sudo-suhas

**Comment by StewartJingga** [comment]
> Should we move sample config to its own file 🤔 ? or even better, maybe we should generate sample config from the `Config` struct itself. wdyt @ravisuhag @sudo-suhas

We can keep current change for now, need more analysis and effort for generating from `Config` struct but I think it is possible. Just something to think about.

**Review by StewartJingga — APPROVED** [review]

File: generator/recipe.go (hunk 1 above)
Comment (sudo-suhas) [reply generator/recipe.go:97]: addressed in https://github.com/odpf/meteor/pull/412, please check

**Pull request merged**